### DISCUSSION TITLE
Custom Error Handling Broke Automatic Documentation.

### DISCUSSION THREAD

**Discussion by BikashShaw** [fastapi/fastapi#7708] [Questions] [answered]
### For our requirements I made the following changes
* [ ] I updated the input validation exception code from 422 to 400.
* [ ] I also modified the default Json error output. 

### My issue
My FastAPI generated automatic documentation is sill showing default error code and error message format.

### My Question
Is it possible to update the API documentation to reflect my change like the correct error code and the right error output format?

**Comment by tiangolo** [comment, marked as answer]
The current way to do it is modifying the generated OpenAPI https://fastapi.tiangolo.com/advanced/extending-openapi/

**Comment by BikashShaw** [comment]
Yeah! It worked for me. Thanks!
